src/patternfly/components/Tabs/__tabs-list.hbs

```diff
@@ -109,6 +111,26 @@ __tabs-list--IsCloseDisabled: bool
         __tabs-item--text="Files"
         }}
     {{/if}}
+    {{#if __tabs-list--IsOverflow}}
+      {{#if __tabs-list--IsOverflowExpanded}}
+        {{> __tabs-item
+          __tabs-item--id="more"
+          __tabs-item--icon-name="more"
+          __tabs-item--aria-label="Show more tabs"
+          __tabs-item--text="More"
+          __tabs-item--IsOverflow="true"
+          __tabs-item--IsOverflowExpanded="true"
+        }}
+      {{else}}
+        {{> __tabs-item
+          __tabs-item--id="more"
+          __tabs-item--icon-name="more"
+          __tabs-item--aria-label="Show more tabs"
+          __tabs-item--text="More"
+          __tabs-item--IsOverflow="true"
+        }}
+      {{/if}}
+    {{/if}}
```

Comment:
fun fact! You can define a non-existent (or empty) `.hbs` partial (we use `{{#> wrapper}}`, which doesn't exist) and set parameters that will be inherited by its children, and since the `.hbs` file doesn't exist/do anything, it doesn't produce extra markup.

Also you can set `__tabs-item--IsOverflowExpanded` conditionally with the bool already set that indicates if its expanded (`__tabs-list--IsOverflowExpanded`)

So you could do something like 

```suggestion
    {{#if __tabs-list--IsOverflow}}
      {{#> wrapper
        __tabs-item--id="more"
        __tabs-item--id="more"
        __tabs-item--icon-name="more"
        __tabs-item--aria-label="Show more tabs"
        __tabs-item--text="More"
        __tabs-item--IsOverflow="true"
        __tabs-item--IsOverflowExpanded=__tabs-list--IsOverflowExpanded
        }}
          {{> __tabs-item}}
        {{/wrapper}}
    {{/if}}
```

Reply:
That is fun!